lisa/schema.py

```diff
@@ -353,9 +353,15 @@ def __hash__(self) -> int:
         return hash(self._get_key())
 
     @staticmethod
-    def create(type: str) -> "FeatureSettings":
+    def create(
+        type: str, extended_schemas: Optional[Dict[Any, Any]] = None
+    ) -> "FeatureSettings":
         # If a feature has no setting, it will return the default settings.
-        return FeatureSettings(type=type)
+        if extended_schemas:
```

Comment:
It doesn't need the check, just call like below.

```python
feature = FeatureSettings(type=type, extended_schemas=extended_schemas)
```

Reply:
if remove the check, will encounter below exception.
```
raise UndefinedParameterError(error_message)
dataclasses_json.undefined.UndefinedParameterError: Received input field with same name as catch-all field: 'extended_schemas': 'None'
```